### ISSUE TITLE
Remark `iframe` is missing a title

### ISSUE THREAD

**Issue by personalizedrefrigerator** [personalizedrefrigerator/jekyll-hematite-theme#2]
# Summary
 * According to [Mozilla's accessibility guidelines](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#use_title_attribute_to_describe_iframe_content), it is important for `iframe`s to both have a `title` attribute and a `title` tag (preferably with the same content).

# Version information
`hematite 0.1.9`
